.github/workflows/tests.yml

```diff
@@ -61,12 +64,13 @@ jobs:
     strategy:
       matrix:
         platform: ["linux", "win32", "darwin"]
+        python-version: ["2.7", "3.6", "3.7", "3.8", "3.9", "3.10"]
       fail-fast: false
     steps:
       - uses: actions/checkout@v2
       - uses: actions/setup-python@v2
       - run: pip install $(grep mypy== requirements-tests-py3.txt)
-      - run: ./tests/mypy_test.py --platform=${{ matrix.platform }}
+      - run: ./tests/mypy_test.py --platform=${{ matrix.platform }} --python-version=${{ matrix.python-version }}
```

Comment:
Maybe we should simplify `mypy_test.py` so that exactly one Python version must be specified?

Reply:
It could be useful to run all versions at once locally, but I'm not terribly married to that.